post_genesis_scripts/create_group/create-group.sh

```diff
@@ -3,12 +3,13 @@ CHAINID="athens_7001-1"
 KEYRING="test"
 HOSTNAME=$(hostname)
 signer="operator"
+node=tcp://3.218.170.198:26657
```

Comment:
Exclude this IP for now. It'll likely change in the future plus we don't want operators to connect to it yet.